src/ruby/ext/symengine/extconf.rb

```diff
@@ -0,0 +1,74 @@
+require 'mkmf'
```

Comment:
Now we install the extension using cmake. The only thing we use rake for is `rake install`. Does `rake install` still generate a Makefile? It doesn't seem to generate it on my machine. So it seems we can remove this file.